cluster/addons/fluentd-elasticsearch/fluentd-es-image/Makefile

```diff
@@ -1,9 +1,11 @@
 .PHONY:	build push
 
+IMAGE = fluentd-elasticsearch
 TAG = 1.3
 
 build:	
-	sudo docker build -t kubernetes/fluentd-elasticsearch:$(TAG) .
+	docker build -t gcr.io/google-containers/$(IMAGE):$(TAG) .
+	docker tag -f gcr.io/google-containers/$(IMAGE):$(TAG) gcr.io/google-containers/$(IMAGE):$(TAG)
 
 push:	
-	sudo docker push kubernetes/fluentd-elasticsearch:$(TAG)
+	gcloud preview docker push gcr.io/google-containers/$(IMAGE):$(TAG)
```

Comment:
The project is determined from the image name/tag; it doesn't need to be passed as a flag. Per [documentation](https://cloud.google.com/tools/container-registry/#pushing_to_the_registry), in order to push to a gcr.io repository, the image has to be tagged `gcr.io/project-id/image-name:tag`. The repository location is `gcr.io/project-id`.